### ISSUE TITLE
Red Matter 2 (2081470)

### ISSUE THREAD

**Issue by Patola** [ValveSoftware/Proton#6106]
# Compatibility Report
- Name of the game with compatibility issues: Red Matter 2
- Steam AppID of the game: 2081470

## System Information
- GPU: RX 6800 XT
- Driver/LLVM version: Mesa 22.1.6-1
- Kernel version: 5.19.1-268-tkg-bmq
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Patola/e51e3af1be3c7d233b8f14fb21be1aa6
- Proton version: 7.0-4 (default)

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

[steam-2081470.log](https://github.com/ValveSoftware/Proton/files/9379026/steam-2081470.log)
## Symptoms 
Game starts, but not in VR -- it starts in the screen, even though it responds perfectly well to the Valve Index knuckles controls.

## Reproduction
1 - Start SteamVR.
2 - Start the game, either via SteamVR Home, or the SteamVR Overlay, or via the SteamVR client pressing play -- the result is the same.
3 - The game will start and respond to the haptic controls, BUT will not appear in VR, only in the screen.

Sending the proton log, maybe also using with `WINEDEBUG=openxr %command%` might help?

**Comment by Patola** [comment]
Well, used `WINEDEBUG=+openxr %command%` and got this log -- interesting line:
`[OVRPlugin][ERROR] ovr_Initialize failed: Unable to load LibOVRRT DLL`
[steam-2081470-WINEDEBUG=+openxr.log](https://github.com/ValveSoftware/Proton/files/9379083/steam-2081470-WINEDEBUG.%2Bopenxr.log)

**Comment by kisak-valve** [comment]
Oculus VR plugin failing to initialize is most likely a red herring.

**Comment by Patola** [comment]
Yeah, thought about that. Tried some test, removed the two OVRPlugin.dll (temporarily, just to test) and the issue was the same, but without the error message on the log.
[steam-2081470.log](https://github.com/ValveSoftware/Proton/files/9381808/steam-2081470.log)

**Comment by Patola** [comment]
Complete proton log without OVRPlugin.dll (previous one was only WINEDEBUG=+openxr)
[steam-2081470.log](https://github.com/ValveSoftware/Proton/files/9381862/steam-2081470.log)

**Comment by Patola** [comment]
I thought that OVRPlugin.dll was openvr plugin. I see now that it's Oculus-only DLL.
Removed OVRPlugin.dll again and tried to start the game. Same thing happens, game starts but only in the screen, except I don't see the error message about "LibOVRRT" again.
Sending the resulting proton log.
[steam-2081470.log](https://github.com/ValveSoftware/Proton/files/9381887/steam-2081470.log)

**Comment by be7b5** [comment]
The same happens for me. Opens on my monitor not in the HMD.

GPU: RX 6800 XT
    Driver/LLVM version: Mesa 22.1.6-1
    Kernel version: 5.19.2-arch1-1-x86_64
    Proton version: Experimental

**Comment by Patola** [comment]
So, what can be done to proceed with this issue? Anything we can do to help? The game seems to "almost be working", I would prefer not to let this bug report become stale. Any tips on what we users can do to help diagnose it, @kisak-valve? Sorry to evoke you, I know you're the busiest. :/

**Comment by lkroll** [comment]
I managed to get the game to display in the headset by running it in dx11 mode using the `-dx11` launch option.
By default it still only shows  the VR grid in the headset, but with this workaround it is playable so far.

**Comment by kisak-valve** [comment]
VKD3D-Proton to OpenXR interop got wired up not too long ago in Proton Experimental and is worth a retest with fresher logs.

**Comment by Patola** [comment]
I'm using another headset now, the Meta Quest 3 -- will sell my Valve Index. Changing to Proton Experimental does not help in any way, but using the `-dx11` flag made it work.
